nix/amazon-corretto-8.nix

```diff
@@ -0,0 +1,57 @@
+{ pkgs }:
+pkgs.stdenv.mkDerivation rec {
+  pname = "amazon-corretto";
+  version = "8";
+
+  src = pkgs.fetchzip {
+    url =
+      "https://corretto.aws/downloads/latest/amazon-corretto-8-x64-linux-jdk.tar.gz";
```

Comment:
Does this file change on updates? If so, the sha will randomly break. We'll need to pin to an exact version to prevent that.